test/flutter_weather_sensor_plugin_test.dart

```diff
@@ -16,8 +16,4 @@ void main() {
   tearDown(() {
     channel.setMockMethodCallHandler(null);
   });
-
```

Comment:
Optional: I think one thing that _might_ be worth adding is some tests, more for documentation than anything.

Reply:
added tests